go.mod

```diff
@@ -55,18 +55,19 @@ require (
 	k8s.io/apimachinery v0.17.1
 	k8s.io/apiserver v0.17.1
 	k8s.io/cli-runtime v0.17.0
-	k8s.io/client-go v0.17.1
+	k8s.io/client-go v8.0.0+incompatible
```

Comment:
Same this one.

Reply:
I do want to point out that this is replaced later at https://github.com/openshift/oc/blob/f44732172c3dcf854f578aabd4465ff7e517ef2d/go.mod#L77. Therefore it does not matter that is it changed at this stage.